solutions/usaco-1060.mdx

```diff
@@ -123,5 +126,31 @@ class Main {
 ```
 				 
 </JavaSection>
+<PySection>
+
+```py
+N = int(input())
+flowers = list(map(int, input().split()))
+ans = 0
+
+def search(i, j):
+	# find average petals in the range i - j
+	avg_petals = sum(flowers[i:j + 1]) / len(flowers[i : j + 1])
```

Comment:
at least be consistent w/ the colon spacing, this i don't have a preference

Reply:
oops